### DISCUSSION TITLE
Can I have title bars for windows in split layout as in tabbed layout?

### DISCUSSION THREAD

**Discussion by gpasques-gh** [i3/i3#6244] [Q&A] [answered]
I would like to have title bars as in tabbed layout but in split layout, at the moment I have my windows with no title bars when there not in split layout. i3 4.22, Debian 12 Bookworm, OG picom, polybar (not sure if that matters), do you know if it's possible to configure it ?

**Comment by okraits** [comment]
You need to set the `default_border` to `normal`:

https://i3wm.org/docs/userguide.html#default_border

**Reply by gpasques-gh** [reply]
I've already done it and nothing changed, maybe I have another software that cut it out ?

**Comment by okraits** [comment]
Title bars might be disabled for some programs using gtk3/4 and client-side decorations. But all other programs not using gtk3/4 should have title bars. You could also remove the `default_border` setting completely because `default_border normal 2` is the default.

**Reply by gpasques-gh** [reply, marked as answer]
I found it, it was a line in the config file : `for_window [class="^.*"] border pixel 1` which I changed to `for_window [class="^.*"] border normal`. Thanks anyway !
